src/Microsoft.ML.FastTree/FastTreeArguments.cs

```diff
@@ -20,7 +20,7 @@ public interface IFastTreeTrainerFactory : IComponentFactory<ITrainer>
     {
     }
 
-    /// <include file='./doc.xml' path='docs/members/member[@name="FastTree"]/*' />
+    /// <include file='doc.xml' path='doc/members/member[@name="FastTree"]/*' />
```

Comment:
❓ Why did this one lose the `./`, but the one in **FastTreeClassification.cs** did not?